Shopify/cli
# Remove semver and upgrade cloudflared

### WHY are these changes introduced?
We are trying to cleanup dependencies that are unused or easily replaceable with custom code.



### WHAT is this pull request doing?
- Remove `semver` as a dependency of `plugin-cloudflare` because we just need to do a simple version comparison.
- Update cloudflared to latest version
- Add a new env var to bypass cloudflare installation.


### How to test your changes?
- If you run `pnpm install` you should see how a new cloudflare binary is intalled. in `plugin-cloudflare/bin`
- If you run it again you should see a message in the log like: `cloudflared already installed, skipping`



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
